geo-types/src/arbitrary.rs

```diff
@@ -0,0 +1,131 @@
+use crate::{
+    CoordFloat, Coordinate, Geometry, GeometryCollection, LineString, MultiLineString, MultiPoint,
+    MultiPolygon, Point, Polygon, Rect, Triangle,
+};
+
+impl<'a, T: arbitrary::Arbitrary<'a> + CoordFloat> arbitrary::Arbitrary<'a>
+    for Coordinate<T>
+{
+    fn arbitrary(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
+        let x = u.arbitrary::<T>()?;
+        if x.is_nan() {
+            return Err(arbitrary::Error::IncorrectFormat);
+        }
```

Comment:
There is value in creating geometries with NaN, but considering we don't have a validity story for `geo`, the NaN values get in the way of fuzz testing

Reply:
> the NaN values get in the way of fuzz testing

Can you expound on what you mean by "get in the way of fuzz testing"? Is it that we `panic` when we plumb through NaN valued Geometries?

Reply:
You know what, let's aim high and include `NaN` for now. If it `NaN` becomes burdensome, we can always filter them out in the future

Reply:
Done in https://github.com/georust/geo/pull/622/commits/ffa3f05eaa581ad78e4a78581a9b66c42f70f8b4